### PR TITLE
refactor: used the same keystore instance to fetch pKey everytime 

### PR DESCRIPTION
**Pull request by Yashk767**
# Description

Creating a new keystore instance for every transaction costed us an opening of separate goroutine which would start a watcher/updater to check the changes in the keystore directory files. Either creating the instance at the start and using the same instance every time  for a transaction would function the same and wouldnt cost up an extra goroutine.

- Created keystore instance at the start and reused the same for every transaction.
- Keystore defined as a field to `AccountManager` struct which is used as a method caller to `GetPrivateKey()`

Fixes https://github.com/razor-network/oracle-node/issues/1197

# How Has This Been Tested?

Ran a staker making transactions in voting process with a goroutine counter placed before and after the getPrivateKey call.

### CODE REVIEW

File: cmd/resetUnstakeLock.go [hunk 1]
```diff
@@ -50,18 +51,22 @@ func (*UtilsStruct) ExecuteExtendLock(flagSet *pflag.FlagSet) {
...
 	stakerId, err := razorUtils.AssignStakerId(flagSet, client, address)
 	utils.CheckError("Error in getting stakerId: ", err)
 
 	extendLockInput := types.ExtendLockInput{
-		Address:  address,
-		Password: password,
 		StakerId: stakerId,
+		Account:  account,
 	}
-	log.Debugf("ExecuteExtendLock: Calling ResetUnstakeLock with arguments extendLockInput = %+v", extendLockInput)
```
Comment (ashish10677) [inline cmd/resetUnstakeLock.go:64]: Why is this log removed.

File: cmd/vote_test.go [hunk 2]
```diff
@@ -275,8 +277,9 @@ func TestCalculateSecret(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			InitializeInterfaces()
-			gotSignature, gotSecret, err := cmdUtils.CalculateSecret(types.Account{Address: tt.args.address,
-				Password: tt.args.password}, tt.args.epoch, testKeystorePath, tt.args.chainId)
+			accountManager := accounts.NewAccountManager(testKeystorePath)
+			account := accounts.InitAccountStruct(tt.args.address, tt.args.password, accountManager)
+			gotSignature, gotSecret, err := cmdUtils.CalculateSecret(account, tt.args.epoch, testKeystorePath, tt.args.chainId)
```
Comment (ashish10677) [inline cmd/vote_test.go:282]: I think the error check should be first, only then gotSignature and gotSecret must be converted to string.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: cmd/claimBounty.go [hunk 3]
```diff
@@ -62,12 +68,10 @@ func (*UtilsStruct) ExecuteClaimBounty(flagSet *pflag.FlagSet) {
 		log.Debug("ExecuteClaimBounty: BountyId: ", bountyId)
 
 		redeemBountyInput := types.RedeemBountyInput{
-			Address:  address,
-			Password: password,
 			BountyId: bountyId,
+			Account:  account,
 		}
 
-		log.Debugf("ExecuteClaimBounty: Calling ClaimBounty() with arguments redeem bounty input: %+v", redeemBountyInput)
```
[the remark this answers is not in the document]
Comment (Yashk767) [reply cmd/claimBounty.go:70]: `reedemBountyLockInput` struct has a password field in it so removed it from the logs while other parameters we do log inside `ClaimBounty()`. Same is being done in other scenarios.

[the remark this answers is not in the document]
Comment (ashish10677) [reply cmd/claimBounty.go:70]: Okay makes sense.

File: cmd/vote_test.go (hunk 2 above)
Comment (Yashk767) [reply cmd/vote_test.go:282]: Done in https://github.com/razor-network/oracle-node/pull/1200/commits/9218d2f3fe64728fd2b022f045798ceab67258e4

**Review by ashish10677 — APPROVED** [review]

**Review by rajkharvar — APPROVED** [review]

**Pull request merged**